google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/cloud/AppEngineStopTask.java

```diff
@@ -42,6 +44,9 @@ public AppEngineStopTask(AppEngineStop stop, String module, String version) {
 
   @Override
   public void execute(ProcessStartListener startListener) {
+    UsageTrackerProvider.getInstance()
```

Comment:
Not available here, but If we really want, we could change the AppEngineStopTask constructor and pass through some info up to here. (I didn't do it ATM.)